js/show-error.js

```diff
@@ -0,0 +1,7 @@
+// Создаем сообщение об ошибке, если данные с сервера не загрузились
+export const showDataError = () => {
+  const dataError = document.createElement('div');
+  dataError.className = 'error-block';
+  dataError.innerHTML = 'Не удалось загрузить данные, попробуйте перезагрузить страницу';
```

Comment:
```suggestion
  dataError.textContent = 'Не удалось загрузить данные, попробуйте перезагрузить страницу';
```
Хотя ты тут контролируешь, что приедет, но всё равно, ты вставляешь тупо текст, так что textContent самое то. Если бы ты делала с тегами, типа с параграфом, тогда другое дело.

https://up.htmlacademy.ru/javascript/25/criteries#b27